### PR TITLE
Test new graph in widget

### PR DESCRIPTION
**Pull request by cpreisinger**

### CODE REVIEW

**Comment by efleming** [comment]
@cpreisinger Would you remove the old Chart stuff also so we can realize the reduction in file size?

File: src/lib/canvas-radar.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    this.renderDataName = this.renderDataName.bind(this)
+    this.renderDataImage = this.renderDataImage.bind(this)
+
+    this.resize = this.resize.bind(this)
+    this.wordWrap = this.wordWrap.bind(this)
+    this.leftTextBound = 0
+    this.rightTextBound = 0
+
+    // TODO: Remove later
+    this.tempReformatData = this.tempReformatData.bind(this)
```
Comment (tomprats) [inline src/lib/canvas-radar.js]: Did you want to remove this or does later mean much later

File: src/lib/canvas-radar.js [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+    }
+    // this.context.canvas.width = 800
+    // this.context.canvas.height = 700
+
+    // Variables for context
+    this.graphStrokeStyle = "#555555"
+    this.graphLineWidth = .5
+
+    // Variables for config
+    // TODO: Make this a config option instead of from the data object
```
Comment (tomprats) [inline src/lib/canvas-radar.js]: Same with this TODO

File: src/lib/canvas-radar.js [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+    let formattedData = []
+
+    for (var i = 0; i < dataLabels.length; i++) {
+      var name = dataLabels[i].text
+      var image = dataLabels[i].image
+      var value = dataValues[i]
+      formattedData.push({name: name, image: image, value: value})
+    }
+
+    // TODO: Remove this block for debugging test data and order of data or make debugging a config option
```
Comment (tomprats) [inline src/lib/canvas-radar.js]: TODO

Comment (tomprats) [inline src/lib/canvas-radar.js]: There's a lot of TODOs. Did you mean to remove them?

**Comment by cpreisinger** [comment]
Lol. I could address some of them. They just don't really affect the functionality. Just some cleanup I wanted to get to when I could. But some I could probably take care of now.

**Pull request merged**